web-services/security/src/main/java/datawave/security/cache/AccumuloCacheStore.java

```diff
@@ -285,8 +283,8 @@ public void purge(Executor threadPool, PurgeListener<? super K> listener) {
     
     @Override
     public void process(KeyFilter<? super K> filter, CacheLoaderTask<K,V> task, Executor executor, boolean fetchValue, boolean fetchMetadata) {
-        try {
-            BatchScanner batchScanner = connector.createBatchScanner(tableName, authorizations, 5);
+        try (BatchScanner batchScanner = connector.createBatchScanner(tableName, authorizations, 5)) {
+            
             batchScanner.setRanges(Collections.singleton(new Range()));
             try {
                 TaskContext taskContext = new TaskContextImpl();
```

Comment:
Can remove `batchScanner.close()` on line 308

Reply:
Fixed in next commit